bin/cheribsdtest/Makefile

```diff
@@ -23,6 +23,10 @@ SUBDIR+=	purecap-benchmark \
 		purecap-benchmark-dynamic \
 		purecap-benchmark-dynamic-mt \
 		purecap-benchmark-mt
+
+.if ${MACHINE_CPUARCH} == "aarch64"
+SUBDIR+=	benchmark-mt-c18n
```

Comment:
You need to tell cheribuild this exists so it actually gets run in CI and we can verify it works